src/App.tsx

```diff
@@ -21,19 +21,20 @@ export default function App() {
         setIsShakeReady={setIsShakeReady}
         hasInitialTransitionRun={hasInitialTransitionRun}
         setHasInitialTransitionRun={setHasInitialTransitionRun}
-        height={height}
       />
     </View>
-  )
-};
+  );
+}
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
+    justifyContent: "center",
+    alignItems: "center",
     paddingHorizontal: 20,
-    backgroundColor: 'black',
-    position: 'relative',
+    backgroundColor: "black",
+    position: "relative",
   },
 });
+
+export default registerRootComponent(App);
```

Comment:
Added this line to manually indicate this file as the app entry for Expo when it runs a build.

Without this line, app entry was being set within `node_modules/expo/AppEntry.js` which was causing build errors.